src/sfizz/Interpolators.hpp

```diff
@@ -48,25 +52,17 @@ class Interpolator<kInterpolatorLinear, R>
 //------------------------------------------------------------------------------
 // Hermite 3rd order, SSE specialization
 
-#if SFIZZ_HAVE_SSE
+#if SIMDE_NATURAL_VECTOR_SIZE_GE(128)
```

Comment:
This should be true for all archs we support, but just in case maybe we can have a generic fallback instantiation for scalar floats (just adding an `#else` with the template instantiation)

Reply:
Wasm lacks this support.
It has already a scalar fallback anyway. I wrote it as a reference.